nautobot/docs/configuration/optional-settings.md

```diff
@@ -960,6 +960,34 @@ If set to `False`, unknown/unrecognized filter parameters will be discarded and
 
 ---
 
+## TEST_FACTORY_SEED
+
++++ 1.5.0
+
+Default: `None`
+
+When [`TEST_USE_FACTORIES`](#test_use_factories) is set to `True`, this configuration provides a fixed seed string for the pseudo-random generator used to populate test data into the database, providing for reproducible randomness across consecutive test runs. If unset, a random seed will be used each time.
+
+---
+
+## TEST_USE_FACTORIES
+
++++ 1.5.0
+
+Default: `False`
+
+If set to `True`, the Nautobot test runner will call `nautobot-server populate_database --flush --seed ...` before executing any test cases, pre-populating the test database with various pseudo-random instances of many models.
+
+!!! warning
+    This functionality requires the installation of the [`factory-boy`](https://pypi.org/project/factory-boy/) Python package, which is present in any Nautobot development environment, but is _not_ an inherent dependency of the Nautobot package when installed otherwise.
```

Comment:
This should be available to any plugin that uses the `nautobot-dev` containers for their development environment, correct? Just if they want to use local testing and it's not in their dev dependency.

Reply:
Ya, but if this were run on a `pip`-installed instance, it wouldn't be there.

Reply:
Correct. But this is strictly a development focused feature so it would affect users who have `nautobot-server test <plugin_module>` running locally instead of a container. Again, not zero but should be considered if they only exclusively develop by running Nautobot & tests in a container.

Reply:
Even then, I'd expect that the plugin *should* declare a wildcard development dependency on `factory-boy`, just like they do for `black`, `pylint`, etc. even though those are also provided by the dev container.